readthedocs/restapi/views/model_views.py

```diff
@@ -94,29 +94,6 @@ class ProjectViewSet(UserSelectViewSet):
     pagination_class = api_utils.ProjectPagination
     filter_fields = ('slug',)
 
-    @decorators.detail_route()
-    def valid_versions(self, request, **kwargs):
```

Comment:
This is deleted because the `project.num_major` and `project.num_minor` are deleted and hence this will always return a `400 BAD REQUEST`.